src/python/WMCore/WMSpec/StdSpecs/StepChain.py

```diff
@@ -445,6 +462,11 @@ def validateSchema(self, schema):
                                                couchDBName=schema["CouchDBName"],
                                                getOutputModules=True)
 
+        if 'KeepOutput' in schema[stepName] and strToBool(schema[stepName]['KeepOutput']) is False:
+            msg = "Dropping the output of the last step is prohibited.\n"
+            msg += "Set the 'KeepOutput' value to True and try again."
+            self.raiseValidationException(msg=msg)
```

Comment:
This is the code of the method:
`logging.error("About to raise exception %s" % msg)`
`raise WMSpecFactoryException(message=msg)`

Should we get rid of of this method and simply raise the WMSpec exception? If so, then I can make it in a new PR, sounds cleaner.